CONTRIBUTING.md

```diff
@@ -254,7 +254,19 @@ scripts\run_backend_tests.bat
 pnpm test
 ```
 
-- Browser tests are located in `js/spa/test` and are defined as `*spec.ts` files. To run browser tests:
+- Browser tests are located in `js/spa/test` and are defined as `*spec.ts` files.
+
+To install browser test dependencies:
+
+```
+pip install -r demo/outbreak_forecast/requirements.txt
+pip install -r demo/stream_video_out/requirements.txt
```

Comment:
We should probably try to remove additional dependencies from the test suite as it slows things down, complicates setup, and introduces additional failure points. Will file an issue.